src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/InputHiddenFieldEnhancedInstanceof.java

```diff
@@ -0,0 +1,31 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+import java.util.Locale;
+
+public class InputHiddenFieldEnhancedInstanceof {
```

Comment:
please guess what is missed over class and in code 

Can we make instance of multliline regexp Check to catch that all Inputs has `// config:` and `// ok` or `// violation` ... all existing violations in code .... add to suppression with comment `utill ....` ?

Reply:
All comments in input files should only be one of three cases:

1. ` // config:`
2. `// ok`
3. `// violation`

Is that correct?

Reply:
yes, and please exclude all Inputs from `grammar` folder, they will not have such comments.

Reply:
multiline comment please

Reply:
Fixed.